### PR TITLE
Billy/fix time in tests

### PR DESCRIPTION
**Pull request by okwme**
tx format got updated
renamed address to be more descriptive as `currentUser `

### CODE REVIEW

**Comment by okwme** [comment]
fixes #34 too?

**Comment by okwme** [comment]
I made UTC a boolean option instead of a string for "timezone" because native date elements don't have a lot of support for parsing timezone information. We'd need to use moment and moment/timezone to parse the string since native Date seems to only have `getTimezoneOffset`. This is however useful with the option I made which will display the time in UTC instead of converting it to the current machine's timezone, when prop `utc` is true.

File: src/components/TmBlock/TmBlock.vue [hunk 1]
```diff
@@ -57,6 +56,10 @@ export default {
     TmLiTransaction
   },
   props: {
+    currentUser: {
```
Comment (jbibla) [inline src/components/TmBlock/TmBlock.vue]: i didn't know you could make props objects with types!

Comment (okwme) [reply src/components/TmBlock/TmBlock.vue]: omg js we should really be doing it all over the place with default values etc 😅

Comment (nylira) [reply src/components/TmBlock/TmBlock.vue]: Yes this is a good idea 👍

**Pull request merged**